src/modelValidator.js

```diff
@@ -74,19 +96,19 @@ class ModelValidator {
   }
 
   /**
-     * Identifier information
-     * @typedef  {Object} IdentifierInfo
-     *
-     * @property {string} identifier - The identifier in question
-     * @property {string} offset - Start location in raw string
-     */
+   * Identifier information
+   * @typedef  {Object} IdentifierInfo
+   *
+   * @property {string} identifier - The identifier in question
+   * @property {string | number} offset - Start location in raw string
```

Comment:
Is this or number, or should it have been just number all along?

Reply:
It is indeed just a number